packages/map/package.json

```diff
@@ -6,9 +6,7 @@
     "dev": "next dev",
     "build": "next build && next export -o build",
     "start": "next start",
-    "lint": "eslint .",
-    "lint:fix": "eslint --fix .",
-    "format": "prettier --write \"**/*.+(js|jsx|json|yml|yaml|css|vue)\"",
```

Comment:
I would also appreciate additional detail here on why `prettier` is being removed altogether. Thanks

Reply:
There were conflicts between eslint and prettier because I forgot to add [configurations](https://nextjs.org/docs/basic-features/eslint#usage-with-prettier). Prettier will be back with the next commit.